lib/main_prod.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+
+import 'app.dart';
+import 'common/utils/env/flavor_config.dart';
+import 'common/utils/logger.dart';
+import 'common/utils/env/environments.dart';
+import 'di/injectable.dart';
+
+// starting the production app
+Future<void> main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  FlavorConfig(
+    flavor: Flavor.prod,
+    name: 'PROD',
+    color: Colors.transparent,
+    values: const FlavorValues(
+      logNetworkInfo: false,
+      showFullErrorMessages: false,
+    ),
+  );
```

Comment:
Not sure how flavorconfig works here. Is it supposed to be saved to a singleton? Could you explain a bit on this?